src/current/v23.1/sso-db-console.md

```diff
@@ -6,94 +6,104 @@ docs_area: manage
 key: sso.html
 ---
 
+CockroachDB clusters allow users to authenticate with Single sign-on (SSO), both to the [DB Console]({% link {{ page.version.version }}/ui-overview.md %}), and for SQL client access, using OpenID connect (OIDC).
```

Comment:
DB Console SSO uses OIDC, the SQL client access just uses JWTs

Reply:
Ok, I think I've got this correctly split out now, please confirm.

Reply:
I'm seeing "Provioning an OIDC client for CockroachDB Cluster SSO authentication, using GCP." 

Is this a typo and is supposed to read DB Console SSO?

also nit: spelling for "provisioning"